src/Appwrite/Utopia/Response/Model/Bucket.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class Bucket extends Model
+{
+
+    public function __construct()
+    {
+        $this
+        ->addRule('$id', [
```

Comment:
Very minor, looks like we use 1 level of indent here . Would be nice to maintain consistency 

https://github.com/appwrite/appwrite/blob/feat-functions-refactor/src/Appwrite/Utopia/Response/Model/Log.php#L13